### ISSUE TITLE
PHP notice on admin page using ACF flexible field (since 2.7)

### ISSUE THREAD

**Issue by mingyeungs** [polylang/polylang#502]
## Prerequisites

- [x] This is not a usage question (Those should be directed to the [community supported forum](https://wordpress.org/support/plugin/polylang), unless this is a question about Polylang Pro in which case you should [use the helpdesk](https://polylang.pro/support/)).
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest `master` branch of Polylang and the latest WordPress version.
- [x] This bug happens with only Polylang plugin active
- [x] This bug happens with a default WordPress theme active
- [x] I can reproduce this bug consistently

## Steps to reproduce the issue

1) Create an **ACF Flexible Field** with two **layouts**, each with one field (with different field name):

- Layout A: field_AA
- Layout B: field_BB

2) Assign the **Flexible Field** to a page, and add 4 rows to that flexible field:

- Layout A [field_AA='hello']
- Layout A [field_AA='world']
- Layout B [field_BB='foo']
- Layout B [field_BB='bar']

3) With PHP error logging enabled, visit the backend page and see 4 notices:
```
Notice: Undefined index: {fieldBB} in /var/www/html/wp-content/plugins/polylang-pro/modules/plugins/acf-sync-metas.php on line 136
Notice: Undefined index: {fieldBB} in /var/www/html/wp-content/plugins/polylang-pro/modules/plugins/acf-sync-metas.php on line 136
Notice: Undefined index: {fieldAA} in /var/www/html/wp-content/plugins/polylang-pro/modules/plugins/acf-sync-metas.php on line 136
Notice: Undefined index: {fieldAA} in /var/www/html/wp-content/plugins/polylang-pro/modules/plugins/acf-sync-metas.php on line 136
```

## Expected behavior and actual behavior

`PLL_ACF_Sync_Metas` tries to read `field_BB` from `Layout A`, and `field_AA` from `layout B`,
raising a lot of PHP Notices.

## I was expecting...

No PHP notice

**Comment by mingyeungs** [comment]
Sorry just noticed it is a Pro version feature. I'll send this to the helpdesk instead.

**Comment by mcaskill** [comment]
Issue also applies to repeater and group fields.

The fix applied in 2.7.2 only addresses flexible content.
